www/app/Http/Controllers/Api/ConversationController.php

```
@@ -337,6 +343,70 @@ public function stream(Request $request, Conversation $conversation): JsonRespon
+            ], 409);
+        }
+
+        if ($this->streamManager->isStreaming($conversation->uuid)) {
+            RequestFlowLogger::endRequest('already_streaming');
+            return response()->json([
+                'success' => false,
+                'error' => 'Conversation is already streaming',
+            ], 409);
+        }
+
+        $this->streamManager->startStream($conversation->uuid, [
```

PHPStan (2.1.46)

[error] Access to an undefined property App\Models\Conversation::$uuid.
Learn more: https://phpstan.org/blog/solving-phpstan-access-to-undefined-property

(property.notFound)